easy-to-use-api/klips-chart-api/content.json

```diff
@@ -0,0 +1,31 @@
+{
+    "params": [
+        [
+            "Parameter",
+            "region",
+            "geomwkt",
+            "threshold"
+        ],
+        [
+            "Mögliche Werte",
+            "dresden, langenfeld",
+            "Punktkoordinate im Koordinatensystem (CRS) WGS84 (EPSG:4326)",
+            "Beliebiger, numerischer Wert"
+        ],
+        [
+            "Beispiel",
+            "dresden",
+            "POINT(13.740893%2051.056977)",
+            "25.5"
+        ],
+        [
+            "Inhalt",
+            "Auswahl einer der beiden Pilotstädte.",
+            "Auswahl der Punktkoordinate, für die die Statistiken ausgegeben werden sollen. Muss in der ausgewählten Region (region-Parameter) liegen.",
+            "Auswahl eines Temperatur-Schwellenwertes, der dargestellt werden soll."
+        ]
+    ],
+    "text": "Definieren Sie unter https://klips-dev.terrestris.de/easy-to-use-api/chart/ eine URL, in der Sie über folgende Parameter Ihre Ausgabe konfigureren können.",
```

Comment:
The URL should be dynamic in the future. If we deploy on another system, e.g test-server the URL would still link to the dev-server. This can be done in a follow-up step and I'll come up with some ideas for that.

Reply:
Thank you! I changed the text in the table but left in the example for now.